test/integration/ggrc/access_control/test_access_control_role.py

```diff
@@ -54,6 +58,46 @@ def test_create(self):
     assert role.delete == 1, \
         "Update permission not correctly saved {}".format(role.delete)
 
+  @staticmethod
+  def _create_objects_mandatory_tests(role_name, mandatory):
+    """Create user, control, role and ACL objects for tests"""
+    with factories.single_commit():
+      user = factories.PersonFactory()
+      control = factories.ControlFactory()
+      role = factories.AccessControlRoleFactory(name=role_name,
+                                                object_type="Control",
+                                                mandatory=mandatory)
+      factories.AccessControlListFactory(person=user,
+                                         ac_role_id=role.id,
+                                         object=control)
+    return {"user": user, "control": control}
+
+  def test_mandatory_delete(self):
```

Comment:
you may make test less if you use ddt here. 
Also you forget to check the DB after import. 


```
@ddt.data(
    {
        "mandatory": True,
        "response": {
            "Control": {
                "row_warnings": {
                    errors.OWNER_MISSING.format(line=3,
                                                column_name=role_name),
                },
            },
        },
    },
    {
        "mandatory": False,
        "response": {},
    },
)
@ddt.unpack
def test_acr_set_empty_via_import(self, mandatory, response):
  """Test set empty via import if acr mandatory is {mandatory}."""
  role_name = "Test role name"
  with factories.single_commit():
    user = factories.PersonFactory()
    control = factories.ControlFactory()
    role = factories.AccessControlRoleFactory(name=role_name,
                                              object_type="Control",
                                              mandatory=mandatory)
    factories.AccessControlListFactory(person=user,
                                       ac_role_id=role.id,
                                       object=control)
  control_id = control.id
  role_id = role.id
  import_data = OrderedDict([("object_type", "Control"),
                             ("Code*", control.slug),
                             (role_name, "--")])
  self._check_csv_response(response, self.import_data(import_data))

  db_data = collections.defaultdict(set)
  for acl in all_models.Control.query.get(control_id).access_control_list:
    db_data[acl.ac_role_id].add(acl.person_id)
  if mandatory:
    # [....] <- your expected content
    self.assertEqual([....], db_data[role_id])
  else:
    self.assertFalse(db_data[role_id])
```